src/clients/client.ts

```diff
@@ -37,7 +37,7 @@ import { EnqueuedTask } from '../enqueued-task'
 class Client {
   config: Config
   httpRequest: HttpRequests
-  tasks: TaskClient
+  tasks: TaskClient<any>
```

Comment:
At the `Client` level, the type is `<any>` as there is no index record that it can be bound to